.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,68 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+# ******** NOTE ********
+
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master, brand-consolidation, production ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '17 23 * * 1'
```

Comment:
This means it would run “At 23:17 on Monday," correct? If so, it would run approximately on Monday evening in North American time, correct? 

![image](https://user-images.githubusercontent.com/6130520/98019353-40ee7980-1dc7-11eb-9624-6f779dabfae6.png)

https://crontab.guru/#17_23_*_*_1

Reply:
yes thats correct. We can change this to anything we want.